src/components/Pagenation.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import icArrow from "../assets/ic_arrow_left.svg";
+import styles from "./Pagenation.module.css";
+
+export default function Pagenation({ page, setPage, pageSize, totalCount }) {
+    const [maxPage, setMaxPage] = useState(1);
+    const [list, setList] = useState([1]);
+
+    useEffect(() => {
+        setMaxPage( totalCount === 0 ? 1 : Math.ceil(totalCount/pageSize) );
+    }, [totalCount, pageSize]);
+
+    useEffect(() => {
+        if (page > maxPage) { setPage(maxPage); return; }
+
+        const temp = Math.ceil(page/5 - 1)*5;
+        const tempList = [temp+1];
+        for (let i = temp+2; i <= Math.min(temp+5, maxPage); i++) { tempList.push(i); }
+        setList(tempList);
+    }, [page, maxPage]);
+
+    return (<div className={styles.pagenation}>
+        <button 
+            onClick={() => setPage( Math.ceil(page/5) === 1 ? 1 : Math.ceil(page/5 - 1)*5 )}
+        >
```

Comment:
💊 제안  
캐러셀과 페이지네이션과 같은 컴포넌트의 경우 prevButton, nextButton 이 특정 조건에서는 disabled 되는 것이 일반적입니다.  
이를 추가해보시면 더 좋을 것 같아요.

Reply:
💊 제안
button type 이 reset이 아니라면 명시해주세요~

```suggestion
        <button 
            type="button"
            onClick={() => setPage( Math.ceil(page/5) === 1 ? 1 : Math.ceil(page/5 - 1)*5 )}
        >
```